electron/package-standalone-release.sh

```diff
@@ -51,12 +51,29 @@ function copy_if_exists {
     fi
 }
 
+function codesign_if_exists {
+     if [ -z "$1" ]; then
+        echo "copy_if_exists requires binary path"
+        exit 1
+    fi
+
+    BIN="${GOX_GUI_OUTPUT_DIR}/${1}"
+
+    if [ -f "$BIN" ]; then
+        echo "signing osx binary"
+        codesign --force --sign "Developer ID Application: yunfei mao" "${BIN}"
```

Comment:
What does `--force` mean?

Reply:
`-force When signing, causes codesign to replace any existing signature on the path(s) given. Without this option, existing signatures will not be replaced`
from the manpages

Reply:
the standalone binary has a signature already ( I dont know here it comes from )
```
Executable=/Users/therealssj/Downloads/skycoin-0.26.0-gui-standalone-osx-darwin-x64/skycoin
Identifier=skycoin-c3ab3afec9fc367ce1c47c00967020e26d883dd4
Format=Mach-O thin (x86_64)
CodeDirectory v=20100 size=146369 flags=0x2(adhoc) hashes=4569+2 location=system
Hash type=sha256 size=32
CandidateCDHash sha1=bff9109e71fb6c4b1660c7df6e3bf08d54cfb903
CandidateCDHash sha256=dca09a2947ff9893b5f23c0c21ed9184662a7130
Hash choices=sha1,sha256
Page size=4096
CDHash=dca09a2947ff9893b5f23c0c21ed9184662a7130
Signature=adhoc
Info.plist=not bound
TeamIdentifier=not set
Sealed Resources=none
Internal requirements count=0 size=12
```
if you try to sign it without `--force`
it returns this error: `skycoin: is already signed`

Reply:
Strange because what motivated me to request this is that when I ran the standalone binary from the website, it warned me that it was from an unknown developer

Reply:
https://developer.apple.com/documentation/security/seccodesignatureflags/kseccodesignatureadhoc
because it has no identity attached to it
`codesign --force --sign - myapp.app"`
this is how adhoc signing is done, by using `-` as a "pseudo" identity